perseo-main/src/test/java/com/telefonica/iot/perseo/RulesServletTest.java

```diff
@@ -83,23 +86,23 @@ public void testDoGet() throws Exception {
      */
     @Test
     public void testDoPost() throws Exception {
-        System.out.println("doPost");
+        logger.info("doPost");
         Server server = Help.getServer(RulesServlet.class);
         server.start();
         try {
             String url = String.format("http://127.0.0.1:%d", Help.PORT);
             JSONObject jr = new JSONObject();
             jr.put("name", "test doPost rule");
-            jr.put("text", Help.ExampleRules[0]);
+            jr.put("text", Help.ExampleRules()[0]);
             Help.Res r = Help.sendPost(url, jr.toString(2));
-            assertEquals(200, r.code);
+            assertEquals(200,r.getCode());
```

Comment:
```suggestion
            assertEquals(200, r.getCode());
```

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823